### DISCUSSION TITLE
Choose terraform scripts language (CDK)

### DISCUSSION THREAD

**Discussion by felixbillon** [microsoft/NubesGen#93] [Ideas]
Hello everyone,

Currently all terraform scripts are written in HCL. Could it be a good idea to support CDK (https://github.com/hashicorp/terraform-cdk) and let user choose language for terraform scripts ?

**Comment by jdubois** [comment]
Unless there's an automatic way to transform HCL to a specific language, there's no way we could achieve this. All our templates are made with HCL, we can't have one version for each language.
